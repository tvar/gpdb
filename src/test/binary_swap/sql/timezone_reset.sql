-- start_ignore
\! gpconfig -r timezone --skipvalidation
-- end_ignore
