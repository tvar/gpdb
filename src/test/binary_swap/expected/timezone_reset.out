-- start_ignore
\! gpconfig -r timezone --skipvalidation
20180424:16:40:14:089987 gpconfig:sdw6:gpadmin-[INFO]:-completed successfully with parameters '-r timezone --skipvalidation'
-- end_ignore
